ml-package-versions-pr.yml

```diff
@@ -0,0 +1,42 @@
+sklearn:
+  models:
+    package: "scikit-learn"
+    minimum: "0.19.2"
+    maximum: "0.23.2"
+    run: |
+      pytest tests/sklearn/test_sklearn_model_export.py --large
+
+  autologging:
+    package: "scikit-learn"
+    minimum: "0.22.2"
+    maximum: "0.23.2"
+    unsupported: ["0.22.2"]
+    requirements: ["matplotlib"]
+    run: |
+      pytest tests/sklearn/test_sklearn_autolog.py --large
+
+  dev:
+    package: "scikit-learn"
+    install: |
+      pip install git+https://github.com/scikit-learn/scikit-learn.git@master
+    requirements: ["matplotlib"]
+    run: |
+      pytest tests/sklearn --large
```

Comment:
Ideally, we should run the same autologging / models test cases against the dev version. This particular `run` statement makes that happen, but it might be better for consistency / correctness going forward if we specify package information at the top of the config & let the `models` and `autologging` sections implicitly reference it.

For example:

```
sklearn:
  package_info:
      pip_release: "scikit-learn"
      pip_dev: "git+https://github.com/scikit-learn/scikit-learn.git@master"

  models:
    minimum: "0.19.2"
    maximum: "0.23.2"
    run: |
      pytest tests/sklearn/test_sklearn_model_export.py --large

  autologging:
    minimum: "0.22.2"
    maximum: "0.23.2"
    unsupported: ["0.22.2"]
    requirements: ["matplotlib"]
```

Reply:
Makes sense. I'll update the config file structure and `generate_matrix.py`.